pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1585,18 +1595,34 @@ func (r *ReconcileInfinispan) serviceExternal(m *infinispanv1.Infinispan) *corev
 			Namespace: m.ObjectMeta.Namespace,
 			Labels:    lsService,
 		},
-		Spec: corev1.ServiceSpec{
+	}
+
+	if externalServiceType == corev1.ServiceTypeLoadBalancer {
```

Comment:
Isn't the `LoadBalancer` type missing ports definition? Especially `targetPort`?

Reply:
yep, tnx!